packages/commons/src/fhir/extension.ts

```diff
@@ -191,12 +191,7 @@ export type KnownExtensionType = StringExtensionType & {
   }
   'http://opencrvs.org/specs/extension/employee-signature': {
     url: 'http://opencrvs.org/specs/extension/employee-signature'
-    valueSignature: {
-      contentType: string
-      when?: string
-      type?: Array<{ system: string; code: string; display: string }>
-      blob: string
-    }
```

Comment:
Aren't we losing some data this way? Like previously we used to store the `contentType` but now we are not. Maybe https://hl7.org/fhir/STU3/datatypes.html#Attachment would be more appropriate?